Automattic/jetpack
# Jetpack: remove video block transform to/from videopress/video for Simple sites

This PR removes the block transform from/to VideoPress video block in dotcom context.

Fixes https://github.com/Automattic/jetpack/issues/27833

#### Changes proposed in this Pull Request:

* Jetpack: remove video block transform to/from videopress/video for Simple sites

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






#### Test on Jetpack/Atomic sites

* Activate Jetpack plugin
* Go to the block editor
* Add/create a core/video block instance
* Confirm you can transform the block to VideoPress video block by using the toolbar

From | To
------|------
<img width="414" alt="Screen Shot 2022-12-08 at 11 39 08" src="https://user-images.githubusercontent.com/77539/206437687-675141b2-3b9e-4ff8-b464-d1c9944ae090.png"> | <img width="348" alt="Screen Shot 2022-12-08 at 11 39 52" src="https://user-images.githubusercontent.com/77539/206437682-c16e3289-9f37-41d8-9551-71ddbd1d9600.png">

#### Test on Simple sites (Sandboxed)

* Go to the block editor
* Add/create a core/video block instance
* Confirm you cannot transform the block to VideoPress video block by using the toolbar. The option is not available.

<img width="370" alt="image" src="https://user-images.githubusercontent.com/77539/206437310-b7363862-fe42-4fee-a7cc-17161d46df96.png">

#### Test on Simple sites (Unsandboxed)

* Confirm ~you can transform the block~ the block transform it nos available (it's a beta extension yet)

<img width="441" alt="image" src="https://user-images.githubusercontent.com/77539/206438815-45c71872-367f-48aa-995e-7ed52ace257f.png">
